examples/datasource-http-backend/cypress/integration/01-smoke.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { e2e } from '@grafana/e2e';
+
+e2e.scenario({
```

Comment:
Guess this was added by create-plugin? Probably a discussion regarding create-plugin, but leaving it here so we don't forget about it. I guess we would like to invite plugin developers to create integration tests and by that provide some boilerplate if it's a panel, a datasource etc?

Reply:
Yes. This was added by create-plugin. I can add some E2E test examples specific for this plugin

Reply:
Not sure. Let's ask the rest of the team what they think, like do we have E2E for the other examples etc.